### DISCUSSION TITLE
Android App

### DISCUSSION THREAD

**Discussion by dmishh** [InternetConnectionMonitor/icm-public#86] [Ideas and Feature requests]
TBD

**Comment by dmishh** [comment]
Comment from @dieterjw 

I would like to have Internet Connection Monitor as an Android app. I haven't found something comparable that is as easy to . I use it simply to monitor the reliability of my Internet connection which is not very good at the moment. A log file would be great as a proof to my provider.

I like the idea to use an old smartphone for this task. I have one lying around and it uses less energy than my PC. My WiFi connection is reliable so I see no additional problems.

Greetings, Dieter

**Comment by ArieKlerk** [comment]
That is exactly what I was trying to do myself today! I thought of using Brave in order to see a life connection status, but couldn't find a way to display it. An Android AND an IOS app would be great! Life Internet Connection App... mmmm....
